README.md

```diff
@@ -199,6 +199,15 @@ nan
 0.0
 ```
 
+### Round()
+
+```python
+>>> round(1.5)
+2
+>>> round(2.5)
+2
```

Comment:
Interesting! The text under "Explanation" immediately below this says to read explanation.md for clarity, but I just read it and it doesn't explain this behavior. Would you be interested in adding an explanation there for this?

Reply:
Sorry for not including the explanation but honestly, I'm not really sure why this happens. The doc mentions nothing about such a case, I might wanna ask someone from core-python regarding this, I guess.

Reply:
@typhonic thanks for clearing that up, I seem to have misinterpreted that. Although I must admit, it seems like a weird choice. I'd add this to explanations.

Reply:
Definitely weird in light of e.g.
```
$ node -e 'console.log(Math.round(2.5))'
3
$ ruby -e 'puts (2.5).round()'
3
$ perl -e 'use Math::Round; print round(2.5)'
3
```
I'm guessing this must have been discussed already on comp.lang.python, but I [didn't immediately see where](https://groups.google.com/forum/#!searchin/comp.lang.python/round$202.5%7Csort:date). Let's find or start a thread to get some more rationale for this and link to it from the explanation.

Reply:
It is a different algorithm than I was taught; however, I have contemporaries (I'm 62) that were taught to round that way. The logic is that x.5 is halfway between x and x+1. In this case, to always round up is an arbitrary direction and can theoretically skew the data.  If you round to the even number, it splits the distribution of direction choices and is more fair. 
Before I had round() I wrote my own formula as "int(x + 0.5)" which was the easy way out.

Reply:
Searched and didn't find anything on python-dev, but just found https://bugs.python.org/issue32956 and asked for rationale there.

Reply:
@typhonic thanks for that insight, although the way I studied round was to always round `x.5` and above to `x+1`, we weren't aware of how it would have affected the data in question and since it was agreed upon by so many other platforms and languages, this rather obscure way of rounding up numbers never caught my eye. Would you happen to have a visual representation of the problem, I'd love to use that in the explanation.

@jab you're right, this must have been discussed somewhere before where there might be a constructive explanation of why this was chosen over the more ubiquitous way of rounding numbers.

Reply:
Just found http://wiki.c2.com/?BankersRounding (thanks @novalis!)

Reply:
It's not a wat in the sense that it is a principled design decision. What I think may very well still be a wat is making this the default behavior of `round(2.5)` (rather than something like `round(2.5, method=HALF_EVEN)` in light of how uncommon this is among other popular languages / how relatively little-known banker's rounding is even among experienced programmers who've even studied applied mathematics in college.

Reply:
@jab you're right, although I'm not sure if this repo is actively maintained anymore, so it's hard to get the repo owner's take on this. Anyway, I came across this recently - https://realpython.com/python-rounding/